File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/Json/JsonLocalizationFile.cs
using System.Collections.Generic;

namespace Volo.Abp.Localization.Json;

public class JsonLocalizationFile
{
    /**
     * Culture name; eg : en , en-us, zh-CN
    */
    public String Culture;// { get; set; }

    public Dictionary<String, String> Texts;// { get; set; }

    public JsonLocalizationFile()
    {
        Texts = new Dictionary<String, String>();
    }
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/json/JsonLocalizationFile.java
package com.ueueo.localization.json;

import lombok.Data;

import java.util.HashMap;
import java.util.Map;

@Data
public class JsonLocalizationFile {
    /**
     * Culture name; eg : en , en-us, zh-CN
     */
    private String culture;

    private Map<String, String> texts;

    public JsonLocalizationFile() {
        texts = new HashMap<>();
    }
}
